docs/UserGuide.adoc

```diff
@@ -522,78 +524,135 @@ Examples:
     names `WikiHow` and `Computer Science`.
-    ** For negative conditions (`nn/`, `nu/`, and `nf/`), bookmarks that match all of the conditions
+    * For negative conditions (`nn/`, `nu/`, and `nf/`), bookmarks that match all of the conditions
     within the same condition type are matched (i.e. AND search). E.g. `nf/Work nf/School` will
     match bookmarks that are not in the folder `Work` and not in the folder `School`.
-    ** Bookmarks that match all conditions across different condition types are matched. E.g. `n/web
+    * Bookmarks that match all conditions across different condition types are matched. E.g. `n/web
     n/mod f/NUS` will match bookmarks with [names that match `web` and/or `mod`] AND [are in the
     folder `NUS`].
-* Note that no bookmarks will be tagged if conditions contradict each other.
-E.g. `u/github.com/mark nu/github.com/`
-****
-
-Examples:
+====
 
-// TODO: add step-by-step screenshots of examples
-* `*autotag* NUS n/NUS n/School n/Uni` +
- Creates an autotag that adds the tag `NUS` to all bookmarks that have names containing
-the keywords `NUS`, `School`, and/or `Uni`.
+Other examples:
 
-* `*autotag* LumiNUS u/luminus.nus.edu.sg nf/Miscellaneous` +
- Creates an autotag that adds the tag `LumiNUS` to all bookmarks with URLs
+// TODO: add screenshots and improve descriptions
+* `*autotag* LumiNUS u/luminus.nus.edu.sg nf/Miscellaneous`
++
+image::ui-screenshots/AutotagCommandExample1.png[]
++
+Creates an autotag named `LumiNUS` which tags all bookmarks with URLs
 containing `luminus.nus.edu.sg` that are not in the folder `Miscellaneous`.
 
-* `*autotag* Quiz f/NUS f/Module nu/github nu/stackoverflow` +
- Creates an autotag that adds the tag `Quiz` to all bookmarks that are either in
- the folder `NUS` or in the folder `Module`, and that do not contain any of the
+* `*autotag* Quiz f/NUS f/Module nu/github nu/stackoverflow`
++
+image::ui-screenshots/AutotagCommandExample2.png[]
++
+Creates an autotag named `Quiz` which tags all bookmarks that are either in
+the folder `NUS` or in the folder `Module`, and that do not contain any of the
 keywords `github` or `stackoverflow` in their URLs.
 // end::autotagcommand[]
 
 ==== Deleting an automatic tag: *`autotag-delete`*
 
-This command deletes a previously created <<autotag, autotag>> from Mark. Tags that are already present will
-not be removed, but no bookmarks will be tagged by this autotag after it is deleted.
+If you no longer need an autotag, you can delete it from Mark using the `autotag-delete`
+command. None of your existing tags are affected when an autotag is deleted.
 
 Format: `*autotag-delete* TAG_NAME`
 
+For example:
+
+* If you want to delete the tag `NUS`, input `*autotag-delete* NUS` into the command box.
++
+image::ui-screenshots/AutotagDeleteCommandUi1.png[]
+
+* Observe that the autotag named `NUS` has been deleted from the autotag panel of the Dashboard.
```

Comment:
Prevent to use command sentence. (from 2101)

Reply:
@kxyoke I changed the format to "You can then check that [result of command, etc.]" but didn't touch the corresponding parts in the annotation sections! Do you want me to edit it, or you'll do it?